components/SidePillTag.vue

```
@@ -0,0 +1,215 @@
+<script setup lang="ts">
+import { ref, computed, onMounted } from 'vue'
+
+const props = withDefaults(defineProps<{
```

ESLint

[error] Insert `⏎··`

(prettier/prettier)